app/views/page/recommended.html.erb

```diff
@@ -0,0 +1,30 @@
+<main class="container">
+  <div class="flex flex-col w-full gap-4">
+    <div class="flex items-center justify-between w-full">
+      <h2 class="text-primary shrink-0">Recommended for you</h2>
+    </div>
+
+    <% if @recommended_talks.present? %>
+      <div class="grid min-w-full grid-cols-1 gap-8 sm:grid-cols-2 md:grid-cols-3 md:[&>:nth-child(4)]:hidden lg:grid-cols-4 lg:[&>:nth-child(4)]:block">
+        <%= render partial: "talks/card", collection: @recommended_talks, as: :talk, cached: true, locals: {back_to: root_path, back_to_title: "Home"} %>
```

Comment:
We might have to pass something along, so that we people can also bookmark talks they get recommended:

<img width="3118" height="1614" alt="CleanShot 2025-10-26 at 00 06 17@2x" src="https://github.com/user-attachments/assets/f5e12196-03a4-4720-a334-475f9b9ccdad" />